pkp/pkp-docs#1398 A11Y - Footer uses nav tag incorrectly

israelcefrin opened the pull request:
Fix the headings and the nav menus in the footer.

kaitlinnewson commented:
Hi @israelcefrin, the PR itself looks fine, but I noticed that having the headings as`h2` in the footer does not make sense in some contexts for logical ordering - for example, on the individual docs pages, where the page title is an `h1` and there are no other `h2` elements.

israelcefrin commented:
Hi @kaitlinnewson ,
I've adjusted the `siteFooter__navHeader` and I also adjusted the color ratio for link hover. Tested on VoiceOver screenreader and test with color contrast check too. It should be more logical and non-intrusive with the page content.

kaitlinnewson commented:
Looks good and the hover colour contrast now meets the guidelines 👍
